config.py

```diff
@@ -1,4 +1,12 @@
-
 DEBUG = False
 
 NWIS_SITE_SERVICE_ENDPOINT = 'https://waterservices.usgs.gov/nwis/site'
+
+# a list of stream gage ids to be included in the map
+SITE_IDS = []
+
+# coordinates for the map's bounding box
```

Comment:
You should be fairly descriptive for the bounding is it (west, north, east, south) or some other combination?